### PR TITLE
ENH Update translations

### PR DESCRIPTION
**Pull request by emteknetnz**
Automated translations update generated using https://github.com/silverstripe/tx-translator

### CODE REVIEW

File: client/lang/src/da.json [hunk 1]
```diff
@@ -35,4 +35,4 @@
     "Admin.VERIFY_TO_CONTINUE": "Bekræft for at forsætte",
     "Admin.ENTER_PASSWORD": "Indtast dit kodeord",
     "Admin.VERIFY": "Bekræft"
-}
+}
```
Comment (ChloeSartorelli) [inline client/lang/src/da.json:38]: It shouldn't be changing whether there's new lines at ends of files I don't think
But that's not a huge deal

File: client/lang/src/en.json [hunk 2]
```diff
@@ -7,7 +7,7 @@
     "Admin.BATCH_UNPUBLISH_PROMPT": "You have {num} page(s) selected.\n\nDo you really want to unpublish",
     "Admin.SELECTONEPAGE": "Please select at least one page",
     "Admin.FormatExample": "Example: {format}",
-    "Admin.NO_SIZE": "N\/A",
+    "Admin.NO_SIZE": "N/A",
```
Comment (ChloeSartorelli) [inline client/lang/src/en.json:10]: Is this correct or is it a serialisation/deserialisation error?

File: lang/en.yml [hunk 3]
```diff
@@ -100,7 +100,7 @@ en:
   SilverStripe\Admin\Navigator\SilverStripeNavigatorItem:
     WONT_BE_SHOWN: 'Note: this message will not be shown to your visitors'
   SilverStripe\Admin\Navigator\SilverStripeNavigatorItem_Unversioned:
-    PREVIEW': 'Preview'
+    "PREVIEW'": Preview
```
Comment (ChloeSartorelli) [inline lang/en.yml:103]: This has highlighted an error in the original yml - should just be `PREVIEW: Preview`

File: client/lang/src/en.json (hunk 2 above)
Comment (emteknetnz) [reply client/lang/src/en.json:10]: I think it's correct and it was wrong before, in the .js file (which is what the .json file is turned into) - it's "N/A" - https://github.com/silverstripe/silverstripe-admin/blob/1/client/lang/en.js#L17

**Comment by GuySartorelli** [comment]
This has a conflict, and doesn't reference an issue... was this an old one that should be closed?

**Pull request closed without merging**